jmix-audit/audit/src/main/java/io/jmix/audit/impl/EntityLogImpl.java

```diff
@@ -443,6 +447,20 @@ protected String findUsername() {
         }
     }
 
+    protected String findSubstitutedUsername() {
+        UserDetails effectiveUser = currentUserSubstitution.getEffectiveUser();
+        if (effectiveUser != null) {
+            return effectiveUser.getUsername();
+        } else {
+            String username = properties.getSystemUsername();
```

Comment:
It is necessary to check how the new column works if changes are made by the system user, for example in the quartz add-on.